libkbfs/kbfs_cr_test.go

```diff
@@ -606,22 +606,16 @@ func TestCRFileConflictWithMoreUpdatesFromOneUser(t *testing.T) {
 		t.Fatalf("Couldn't sync file: %v", err)
 	}
 
-	// User 2 makes a new different file
-	data2 := []byte{5, 4, 3, 2, 1}
-	err = kbfsOps2.Write(ctx, fileB2, data2, 0)
-	if err != nil {
-		t.Fatalf("Couldn't write file: %v", err)
-	}
+	// User 2 makes a few changes in the file
+	for i := byte(0); i < 4; i++ {
+		// This makes sure the unmerged head of user 2 is ahead of (has large
+		// revision than) the merged master branch.
```

Comment:
Please add a little more explanation here, something like: "... the merged master branch, so we can test that we properly fetch updates when the unmerged revision is greater than the merged revision (regression for KBFS-1206)."